code/script.js

```diff
@@ -19,33 +121,36 @@ const showMessage = (message, sender) => {
       </section>
     `
     // the else if statement checks if the sender is a bot and if that's the case it inserts an html senction inside the chat with the posted message
+
   } else if (sender === 'bot') {
+    console.log("Hiya!")
     chat.innerHTML += `
       <section class="bot-msg">
-        <img src="assets/bot.png" alt="Bot" />
+        <img src="assets/pizzaman.png" alt="Bot" />
         <div class="bubble bot-bubble">
           <p>${message}</p>
         </div>
       </section>
     `
   }
   // This little thing makes the chat scroll to the last message when there are too many to be shown in the chat box
+
   chat.scrollTop = chat.scrollHeight
 }
 
 // Starts here
+
 const greetUser = () => {
+
   // here we call the function showMessage, that we declared earlier with the argument "Hello there, What's your name?" for message, and the argument "bot" for sender
-  showMessage("Hello there, What's your name?", 'bot')
-  // Just to check it out, change 'bot' to 'user' here 👆
+
+  showMessage("Ciao, What's your name?", 'bot')
+  
 }
 
 // Set up your eventlisteners here
+// Added a eventlistener that invoke the function sendMessage
+
+form.addEventListener('submit', sendMessage);
 
-// When website loaded, chatbot asks first question.
-// normally we would invoke a function like this:
-// greeting()
-// But if we want to add a little delay to it, we can wrap it in a setTimeout:
-// setTimeout(functionName, timeToWaitInMilliSeconds)
-// This means the greeting function will be called one second after the website is loaded.
-setTimeout(greetUser, 1000)
+setTimeout(greetUser, 700);
```

Comment:
The README file is not updated. I can’t see where or if the code is deployed somewhere like Netlify? There is no description of how you chose to take on the task, any problems you encountered, etc.